_sass/layout/_l-banner.scss

```diff
@@ -45,8 +45,16 @@ $backgrounds: 'promoHome', 'promoInteriores-About', 'promoInteriores-CompareCoun
 
     > .wrap {
       display: flex;
-      justify-content: center;
-      flex-direction: column;
+      justify-content: space-between;
+      flex-direction: row;
```

Comment:
Be extremely careful when doing changes in this general elements.  You might be affecting more elements.
Be sure you are not crashing another elements and if so, make and exception (&.-my-awesome-exception)

Hint: Check one index page.